testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -83,37 +91,125 @@ Any project that has the following Gems enabled:
+| Creating an entity with PhysX Character Controller component    | <ol><li>Create an entity with PhysX Character Controller based on each supported shape, add a mesh component to it for visibility.<li>Collide other PhysX entities with PhysX Character Controller entity to verify its collision.</ul></li><li>Verify behavior of collisions with PhysX Character Controller after assigning it with various Physics Materials.</li></ol> | <ul><li>PhysX character controller as a kinematic controller is not affected by gravity and outside forces but can still collide with other PhysX entities.</li><li>PhysX character controller can be assigned with all available shapes and these shapes dictate how other entities collide with PhysX Character Controller.</li><li>Shapes assigned to PhysX Character Controller can be modified with options found in the component.</li></ul>
+| Adding PhysX Character Gameplay component to PhysX Character Controller entity | <ol><li>Add a PhysX Character Gameplay component to PhysX Character Controller entity.</li><li>Change component values of PhysX Character Gameplay.</li><li>Enter Game Mode when PhysX Character Controller entity with PhysX Character Gameplay component is present in the Viewport.</li></ol>                                                                                                                                   | <ul><li>PhysX Character Gameplay adds non-kinematic game logic properties to PhysX Character Controller such as gravity.</li><li>Modifying component values found inside PhysX Character Gameplay impacts behavior of PhysX Character Controller entity during simulation.</li></ul>                                                                                                                                                                                                                                                                                                                         |
+---
+
 ### Area: PhysX Joints
 
 **Project Requirements**
 
-Any project that has the following Gems enabled: 
-* PhysX 
-* Primitive Assets 
+Any project that has the following Gems enabled:
+
+* PhysX
+* Primitive Assets
 * Script Canvas Physics
 
 **Platforms:**
 * Windows
 * Linux
 
-**Docs:** 
+**Docs:**
+
 * [O3DE PhysX Ball Joint](https://www.o3de.org/docs/user-guide/components/reference/physx/ball-joint/)
 * [O3DE PhysX Fixed Joint](https://www.o3de.org/docs/user-guide/components/reference/physx/fixed-joint/)
 * [O3DE PhysX Hinge Joint](https://www.o3de.org/docs/user-guide/components/reference/physx/hinge-joint/)
 
-**Product:** A scene with PhysX Joint entities that test their ability to interact with a scene
+**Product:** A scene with PhysX Joint entities that test their ability to interact with a scene.
 
-**Suggested Time Box:** 60 minutes
+**Suggested Time Box:** 40 minutes
```

Comment:
Does 40 minutes leave any time left over for experimentation? If not, add in a ~5-10 minutes to allow for that.